autoupdate.services/test/unit/src/org/netbeans/api/autoupdate/UpdateProviderFactoryCreateTest.java

```diff
@@ -50,7 +50,7 @@ protected void tearDown () throws  Exception {
     public void testCreate () throws Exception {
         String name = "new-one";
         String displayName = "Newone";
-        URL url = UpdateUnitFactoryTest.class.getResource ("data/catalog.xml");
+        URL url = CatalogCacheTest.class.getResource ("data/catalog.xml");
```

Comment:
Why not use `UpdateProviderFactoryCreateTest.class` here? Or even `getClass().getResource()` ?

Reply:
Fixed